config/database.js

```diff
@@ -1,18 +1,62 @@
+const UserRights = require("./userRights.js");
 
-exports.isLoggedIn = function(req, res, next) {
-		if (process.env.NODE_ENV === "test"){
-			return next();
-		}
-	    if (req.isAuthenticated())
-	        return next();
+/**
+ * A function that validates whether the user is logged in, and if not redirect them to the log in page.
+ * @param req
+ * @param res
+ * @param next
+ * @returns {*}
+ */
+exports.isLoggedIn = function (req, res, next) {
+	if (process.env.NODE_ENV === "test") {
+		return next();
+	}
+	if (req.isAuthenticated())
+		return next();
+
+	res.redirect('/');
+};
 
-	    res.redirect('/');
+/**
+ * A helper function to check if all rights in an array are satisfied.
```

Comment:
Perhaps this needs to be in another file? The rights are not really concerned with the database.

Reply:
I agree that the added code isn't directly related to the database. However, nothing in the file is related to the database. It's all authorization and authentication. I believe it should be in this file but the file really needs to be renamed. Could you do that?

Reply:
Yes ill rename it to authorization